pkg/pod/container.go

```diff
@@ -259,3 +334,46 @@ func areCapabilitiesValid(capabilities []string) bool {
 
 	return valid
 }
+
+func uniqueCaps(sCap []v1.Capability) []v1.Capability {
```

Comment:
missing a description for the function

Reply:
yup, just saw other functions without description so _followed  the pattern_ 😄  thanks, updated